components/Tabs/Tabs.js

```diff
@@ -1,58 +1,63 @@
 class TabLink {
-  constructor(element){
+  constructor(element) {
     // assign this.element to the element reference
-    this.element;
+    this.element = element;
     // Get the tab data attribute and save the value here
-    this.tabData = ; 
+    this.tabData = this.element.dataset.tab;
     // Find all elements with the .card class in index.html that correspond to the tab data attribute
+    this.cards = document.querySelectorAll(".card");
```

Comment:
This was only meant to be in the if/else statement. I understand the confusion in the instruction however.